packages/react-microlink/src/components/Card/CardMedia/video.js

```diff
@@ -39,16 +39,23 @@ const Video = styled.video`
   bottom: 0;
   left: 0;
   z-index: 100;
+
+  ${({autoplay}) => autoplay && `
+    &::-webkit-media-controls-start-playback-button {
+      display: none!important;
+      -webkit-appearance: none;
+    }
+  `}
 `
 
-const CardVideo = ({cardSize, video, image, muted, autoPlay, loop}) => {
+const CardVideo = ({cardSize, video, image, muted, autoplay, loop}) => {
   return (
     <VideoWrapper cardSize={cardSize}>
       <Video
         src={video}
         poster={image}
         muted={muted}
-        autoPlay={autoPlay}
+        autoplay={autoplay}
```

Comment:
I think react says need to be `autoPlay`

Reply:
Whoops of course, camelCase props 🤦‍♂️

Reply:
Weirdly, `playsinline` works 🤔